charms/worker/k8s/charmcraft.yaml

```diff
@@ -75,12 +75,12 @@ config:
         Example:
           e.g.: key1=value1 key2=value2
     bootstrap-datastore:
-      default: dqlite
+      default: managed-etcd
```

Comment:
This change means that any deployment currently installed with `dqlite` set by default (not explicitly set) will upgrade and have its config changed to `managed-etcd`. 

This new default is RIGHT -- but will cause upgrades to be blocked until the user actively sets their cluster datastore with 

```sh
juju config k8s bootstrap-datastore='dqlite'
```